effectiveOfficeBackend/Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM gradle:7-jdk11 AS build
+FROM gradle:7-jdk11 AS graalvm-install
```

Comment:
What is this stage used for? It performs some steps to download graalvm and is not used in any of the final layers.